### PR TITLE
Update Governance and Contributing doc

### PR DESCRIPTION
**Pull request by magdmartin**
See discussion in #2292

### CODE REVIEW

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:137]: Fix grammar.
Isn't contribution really to be worded as **donation?**
```suggestion
Combined with CS&S, the Steering and Advisory committees have experience applying and managing grants. If you are interested in applying to a grant to improve OpenRefine, please share your idea early on the Developer mailing list. It will be a pleasure helping you through your grant application. New grants must be approved by the Advisory Committee. that CS&S retains a 15% handling fees on any donation to finance their operations (20% for government funding). 
```

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:141]:
```suggestion
CS&S owns and manages the domain openrefine.org 
```

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:95]:
```suggestion
It is composed of at least three members. No more than 49 percent of the members of the Advisory Committee may be paid by the project.
```

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:102]:
```suggestion
* Participate in at least one call per quarter;
```

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:105]: Hmm, I don't think there needs to be an explicit requirement that advisory committee members MUST be part of the Admin team.  But if others disagree, that's fine. We just need to get consensus on what qualifies to be an Admin for the GitHub project or not.  Perhaps Roles and Responsibilities on GitHub can be put into a new section.  I don't know.

```suggestion
* Can be part of the Admin team for the project on GitHub
```

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:61]: They might not do it themselves but use a backup or designated peer, like the current situation with Tom and Antonin, and we should not have wording that does not allow backup managers. So I suggest a more open rephrasing of:
```suggestion
* Prepare and coordinate publishing the release 
```

**Review by thadguidry — CHANGES_REQUESTED** [review]

File: GOVERNANCE.md
Comment (magdmartin) [reply GOVERNANCE.md:105]: Good point @thadguidry  - In that case I will remove the mention of the GitHub admin team entirely.

File: GOVERNANCE.md
Comment (magdmartin) [reply GOVERNANCE.md:25]: New suggestion: 
> Guest Contributors are new users getting familiar with the projects or users that have not expressed they interest to be involved with the Organization.

**Comment by thadguidry** [comment]
> 
> 
> I see the following usage of the GitHub group
> 
>     * Admin = Advisory Committee
> 
>     * Contributor =  Org member Contributors
> 
>     * Maintainers = currently not used.
> 
>     * Outside Collaborators =  used for Weblate integration

I think we need to completely remove the mention of Advisory Committee from GitHub operations.
I think what we are really trying to frame up inside the Governance document is that:
- The role **Admin** on GitHub is a privileged role
- We want to have a documented decision process for voting folks into that **Admin** role
- In general, I think we want to say something like other projects do (take a look at the entire page for the [Rust Compiler Team](https://rustc-dev-guide.rust-lang.org/compiler-team.html) or [Node.js](https://nodejs.org/en/get-involved/contribute/) for a good guideline):

I really like this note from Rustc Compiler Team:
> Membership is both a recognition but also an obligation: compiler team members are generally expected to help with upkeep as well as doing reviews and other work.

I think something like this should be added somewhere:
  - > Individuals identified by the OpenRefine Steering/Advisory Committee who have made significant and valuable contributions to OpenRefine and its design may be added to the Admin team or Core Contributors team and given commit access to the project.

Furthermore, I personally really like the activities that Node.js lists and that we could borrow from
> Activities taken into consideration include (but are not limited to) the quality of:
> 
>     code commits and pull requests
>     documentation commits and pull requests
>     comments on issues and pull requests
>     contributions to the Node.js website
>     assistance provided to end users and novice contributors
>     participation in Working Groups
>     other participation in the wider Node.js community
> 
> If individuals making valuable contributions do not believe they have been considered for commit access, they may log an issue or contact a TSC member directly.

File: GOVERNANCE.md
Comment (thadguidry) [reply GOVERNANCE.md:25]: I don't know if I would even put anything about Contributors in the GOVERNANCE.md doc.  I think maybe this section can go into the CONTRIBUTING.md doc with my further reply notes about the decision process, etc.

Here in the GOVERNANCE.md doc, I think we can just mention that the Advisory or Steering committee have the responsibility to vote to allow someone to be added to the GitHub Admin team or Core Contributors team.  Something simple 1 line about that vote responsibility.  And the rest of Roles, Access, GitHub, Contributing, can all go into CONTRIBUTING.md  I.E. stick to only Governance matters in the GOVERNANCE.md doc.

Comment (magdmartin) [reply GOVERNANCE.md:25]: The goal of the governance document is to present the different roles and responsibilities in the project. It is the right place to explain the difference between a Guest Contributor (with no access to manage the repo) and a Contributor (who can manage the repo)

**Comment by magdmartin** [comment]
ping @OpenRefine/contributors for your feedback on this draft

**Review by wetneb — COMMENTED** [review]
Thanks for updating this. I agree there are still open questions about how we renew the list of people in each of those categories. We need to have discussions about this on the mailing lists and ask around for advice (for instance by looking at how it is done in other CS&S projects).

At the moment, only GitHub users who are in the admin group of the OpenRefine org can approve PRs and merge them. So as things stand this group should probably be mentioned in this document (and we need to think more about how people get in and out of this group).

Currently we try to invite contributors to the organization pretty early on, but that does not give them a lot of rights on the project though, they can essentially only tag issues (and wear an OpenRefine badge on their profile, which I guess adds some recognition). We need to think about how to make them progress to other responsibilities later on.

For this PR, I think we need to decide whether the PR should:
* update this document to reflect the current gouvernance, with all the holes we currently have in it (how do we elect people to positions?). In this case we should replace the ``// TO DO` by formulations that are a bit cleaner (making it clear that we want to change them soon).
* or aim to only merge this once we are satisfied by the gouvernance model it describes. That would be nicer, but might take longer.

File: GOVERNANCE.md
Comment (tfmorris) [reply GOVERNANCE.md:40]: Yes, agreed. Please update to match wetneb's comment about the current practice

**Comment by tfmorris** [comment]
The concept of "guest contributor" doesn't make much sense to me. I see basically two levels: contributors & committers. Admin should be separate from code-related stuff and small, with just a few day-to-day admins and a failsafe backup admin like CS&S.

Github has a much more fine grained permission structure than it did originally, so we should make use of those finegrained permissions to allow a bigger group of committers, for those who have demonstrated merit, while still keeping the list of people who can delete/rename repos and do other destructive things small.

As an aside, since I'm an organization owner, I have admin privs even though I'm not part of of the `admins` team. I guess I can add myself to make that more transparent.

**Comment by magdmartin** [comment]
I think it will more effective to discuss and resolve them in a call. Who
would like to participate ?

I think it would be more effective to discuss proposed changes in a call.
I created a doodle to gather availability (between Jan 25 and 30 2021)
https://doodle.com/poll/zisrtcbqpa7e6ggk?utm_source=poll&utm_medium=link

**Comment by magdmartin** [comment]
I will close the doodle pool in 3h from now.

*--*
*Martin Magdinier*
+1 (514) 559 4563

**Comment by magdmartin** [comment]
The call is scheduled for Wed Jan 27 at  5PM UTC. You can join using this
link: https://meet.google.com/fnn-zgra-ado



*--*
*Martin Magdinier*
+1 (514) 559 4563

**Comment by magdmartin** [comment]
I updated the document with today's discussion. 

Next:
- Define how we are part of the advisory committee
- Define how we name the release manager 
- Clarify who is Github owner how they are added/removed. 
- Update the GitHub permission and especially the [team/contributors](https://github.com/orgs/OpenRefine/teams/contributors)  to reflect this document

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:36]: 2 m's and 2 t's.  So change to `Committers` throughout the doc.

**Review by thadguidry — CHANGES_REQUESTED** [review]

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:37]: However, I see this mention of "committer" as unnecessary.  I think we are using the wrong term here.
Commit is a `git` thing.
Contribution is a Project health metric we want to entertain and invite much more.

File: GOVERNANCE.md
Comment (thadguidry) [inline GOVERNANCE.md:39]: We should use the terms **Project Admins**, **Project Owners** and **Contributors**.
But I caution on even adding Project Owners into this Governance because we want Contributors to actually feel like Owners, being responsible for the health of the project and taking an active role and continuing in its development.  So, to that end, I really think we can simplify with only 2 levels...**Project Admins**, **Contributors** ?

Comment (magdmartin) [reply GOVERNANCE.md:39]: From @tfmorris  comment #3198#issuecomment-721289810, and in today's call I understood we agreed to contributors & committers.   I really like to close this PR and move to the next steps. 
If we can't get an agreement on the terminology by next Monday I ask for a vote on the dev mailing list (using a survey).

Comment (thadguidry) [reply GOVERNANCE.md:39]: I am OK with the current terminology.

**Review by wetneb — APPROVED** [review]
I will open a follow-up PR to tidy up a few things, but for now this is a clear improvement over the previous state of this document, so let's get this finally merged.

File: GOVERNANCE.md
Comment (wetneb) [reply GOVERNANCE.md]: This is still not clear enough but I'll open another PR to get this right.

**Pull request merged**
